testsuite/kuadrant/policy/authorization/sections.py

```diff
@@ -138,7 +138,7 @@ def add_api_key(
         name,
         selector: Selector,
         *,
-        all_namespaces: bool = False,
+        all_namespaces: bool = True,
```

Comment:
I dont understand the change. Cant you just change it in `test_all_namespaces_api_key.py` test only instead?

Reply:
Kuadrant creates all AuthConfigs in the namespace where Authorino is running. In our case, they are moved from `kuadrant` to `kuadrant-system` namespaces.
I need either force authorino to look into all namespaces (like I did here), or create all API key secrets in `kuadrant-system` namespaces. What is better?